hs_core/models.py

```diff
@@ -814,6 +814,41 @@ def validate_identifiers(cls, identifiers):
                                                    f"\'{id_link}\' is not a valid {id_name}.")
         return identifiers
 
+    def to_json(self):
```

Comment:
I'm a bit concerned about spreading the conversion across the codebase within each element like this. I get that is what we are doing with the rdf triples work, but since this isn't going across all the metadata, I kind of would like everything done in one place so it is clear and separate.

Reply:
The metadata-extractor already does the transformation of the json for us. By working with the json instead of the django models, I think it'll be more transparent what the transformations actually are. https://github.com/CUAHSI/metadata-extractor/blob/main/hsextract/adapters/hydroshare.py#L292

Reply:
@sblack-usu  So we create the json from db metadata models and write the json file to disk without the schema.org transformation?